commands/raids/report-privacy.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const log = require('loglevel').getLogger('AutoCommand'),
```

Comment:
Change the logger name to PrivacyCommand or somesuch to go along with....

Reply:
D'oh.